internal/cmd/schema-registry/command_schema_create.go

```diff
@@ -113,3 +151,16 @@ func (c *command) schemaCreate(cmd *cobra.Command, _ []string) error {
 	output.Printf(errors.RegisteredSchemaMsg, response.Id)
 	return nil
 }
+
+func read(path string, v any) error {
+	if path != "" {
```

Comment:
You can avoid extra nesting if you short-circuit the function at the beginning.
```suggestion
	if path == "" {
		return nil
	}
```

Reply:
(Also, would it make more sense if we check for `""` before we call this function?)

Reply:
I think checking for `""` inside the function shorter the code.